GPIO test code tests/drivers/gpio/gpio_basic_api does not compile for microchip board mec15xxevb_assy6853

Describe the bug
Tried to compile the standalone gpio test code for microchip and it gives compilation error.
What have you tried to diagnose or workaround this issue?
Adding the line #define DT_ALIAS_GPIO_0_LABEL DT_MICROCHIP_XEC_GPIO_40081000_LABEL
in the file zephyr/tests/drivers/gpio/gpio_basic_api/src/test_gpio.h solves the issue. But this has to be added for all ports and in the proper way in the zephyr stack.

To Reproduce
Steps to reproduce the behavior:
1.	Cd zephyrproject/zephyr/tests/drivers/gpio/gpio_basic_api/
2.	West build -p -b mec15xxevb_assy6853 .
2.	See error

Expected behavior
Should compile without errors and give the output binary.

**Impact**
Build should succeed. It affects gpio testing for microchip.

Screenshots or console output
In file included from ../src/test_callback_manage.c:15:
../src/test_gpio.h:22:2: error: #error Unsupported board
 #error Unsupported board
  ^~~~~
../src/test_callback_manage.c: In function 'test_callback_add_remove':
../src/test_callback_manage.c:84:42: error: 'DEV_NAME' undeclared (first use in this function)
  struct device *dev = device_get_binding(DEV_NAME);
                                          ^~~~~~~~
../src/test_callback_manage.c:84:42: note: each undeclared identifier is reported only once for each function it appears in
../src/test_callback_manage.c: In function 'test_callback_self_remove':
../src/test_callback_manage.c:126:42: error: 'DEV_NAME' undeclared (first use in this function)
  struct device *dev = device_get_binding(DEV_NAME);
                                          ^~~~~~~~
../src/test_callback_manage.c: In function 'test_callback_enable_disable':
../src/test_callback_manage.c:171:42: error: 'DEV_NAME' undeclared (first use in this function)
  struct device *dev = device_get_binding(DEV_NAME);
                                          ^~~~~~~~
ninja: build stopped: subcommand failed.



Environment (please complete the following information):
1.	OS : Ubuntu LTS 18.04
2.	Toolchain: Zephyr SDK : 0.10.3
3.	Zephyr version: v2.1.0-1048-gdbdfd2995b2b 




**Additional context**

--- a/tests/drivers/gpio/gpio_basic_api/README.txt
+++ b/tests/drivers/gpio/gpio_basic_api/README.txt
@@ -1,0 +1,21 @@
+GPIO 2-Pin Test
+###############
+
+This application tests the GPIO subsystem using a hardware configuration
+where two GPIOs are directly wired together.  The test pins are
+identified through a test-specific devicetree binding in the `dts/`
+subdirectory, implemented for specific boards by overlay files in the
+`boards/` directory.
+
+For some boards `src/main.c` may also need to be extended to configure
+the selected pins for use as GPIOs.
+
+Only boards for which an overlay is present can pass this test.  Boards
+without an overlay, or for which the required wiring is not provided,
+will fail with an error like this:
+
+    Validate device GPIO_0
+    Check GPIO_0 output 2 connected to input 3
+    FATAL output pin not wired to input pin? (out high => in low)
+
+No special build options are required to make use of the overlay.

